index.bs

```diff
@@ -577,66 +577,155 @@ urlPrefix: https://w3c.github.io/FileAPI/#dfn-; type: dfn;
 		<div class="algorithm" data-algorithm="navigator-clipboard">
 		<h4 id="h-navigator-clipboard"><dfn>clipboard</dfn></h4>
 		The [=clipboard=] attribute must return the {{Navigator}}'s [=clipboard=] object.
+		It is used to execute read/write operations from/to the [=system clipboard=].
 		</div>
 
 		</div><!-- dfn-for Navigator -->
 
-	<h3 id="clipboard-interface">Clipboard Interface</h3>
-
+	<h3 id="clipboard-item-interface">ClipboardItem Interface</h3>
 		<pre class="idl" data-highlight="webidl">
-		typedef sequence&lt;ClipboardItem> ClipboardItems;
+			typedef (DOMString or Blob) ClipboardItemDataType;
+			typedef Promise&lt;ClipboardItemDataType&gt; ClipboardItemData;
 
-		[SecureContext, Exposed=Window] interface Clipboard : EventTarget {
-		  Promise&lt;ClipboardItems> read();
-		  Promise&lt;DOMString> readText();
-		  Promise&lt;undefined> write(ClipboardItems data);
-		  Promise&lt;undefined> writeText(DOMString data);
-		};
+			callback ClipboardItemDelayedCallback = ClipboardItemData ();
 
-		typedef (DOMString or Blob) ClipboardItemDataType;
-		typedef Promise&lt;ClipboardItemDataType> ClipboardItemData;
+			[SecureContext, Exposed=Window] interface ClipboardItem {
+			constructor(record&lt;DOMString, ClipboardItemData> items,
+				optional ClipboardItemOptions options = {});
 
-		callback ClipboardItemDelayedCallback = ClipboardItemData ();
+			readonly attribute PresentationStyle presentationStyle;
+			readonly attribute FrozenArray&lt;DOMString> types;
 
-		[Exposed=Window] interface ClipboardItem {
-		  constructor(record&lt;DOMString, ClipboardItemData> items,
-		    optional ClipboardItemOptions options = {});
-		  static ClipboardItem createDelayed(
-		      record&lt;DOMString, ClipboardItemDelayedCallback> items,
-		      optional ClipboardItemOptions options = {});
+			Promise&lt;Blob&gt; getType(DOMString type);
+			};
 
-		  readonly attribute PresentationStyle presentationStyle;
-		  readonly attribute long long lastModified;
-		  readonly attribute boolean delayed;
+			enum PresentationStyle { "unspecified", "inline", "attachment" };
 
-		  readonly attribute FrozenArray&lt;DOMString> types;
+			dictionary ClipboardItemOptions {
+			PresentationStyle presentationStyle = "unspecified";
+			};
+		</pre>
+		<p><dfn export for=ClipboardItem id=concept-clipboard-item>ClipboardItem</dfn></p>
+		A [=ClipboardItem=] is conceptually data that the user has expressed a desire to make shareable by invoking a "cut" or "copy" command.
```

Comment:
So you seem to have introduced a separate "conceptual" `[=ClipboardItem=]`, which is not the same as a `{{ClipboardItem}}` IDL/JavaScript object. (Note: normally we would give these "conceptual" items names like "clipboard item", not "ClipboardItem".)

That can work, but you need to be explicit about the relationship. E.g., does each `{{ClipboardItem}}` have a `[=ClipboardItem=]` (which you might call `the {{ClipboardItem}}'s [=ClipboardItem/clipboard item=]?). That is what is done for things like `Response`'s response in https://fetch.spec.whatwg.org/#concept-response-response . Then you have to be very clear about what acts on the conceptual clipboard item, and what acts on the `{{ClipboardItem}}` object. E.g. below I suggest associating a "presentation style" with a `{{ClipboardItem}}`. But maybe it should instead be associated with the conceptual clipboard item, if it's something the OS cares about? Then the getter steps would do something like `Return [=this=]'s [=ClipboardItem/clipboard item=]'s [=clipboard item/presentation style=]`.

Or, you could try to directly store information on the `{{ClipboardItem}}` object. That is probably the more common approach, but it might get confusing when talking about the OS integration, since I assume operating systems don't directly deal with IDL/JS objects.

Reply:
Added `[=clipboard item=]` and its relationship with `{{ClipboardItem}}`.